release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1268,6 +1268,19 @@ In addition to the OCI image, the image-based upgrade uses the `ostree` library
 
 For more information, see xref:../edge_computing/image_based_upgrade/cnf-understanding-image-based-upgrade.adoc#understanding-image-based-upgrade-for-sno[Understanding the image-based upgrade for {sno} clusters].
 
+[id="ocp-4-16-image-based-upgrade-enhancements_{context}"]
+==== Image-based upgrade enhancements
+
+With this release, the image-based upgrade introduces the following enhancements:
+
+* Simplifies the upgrade process for a large group of managed clusters by adding the `ImageBasedGroupUpgrade` API on the hub
```

Comment:
Consider adding a period to each sentence given the length of each sentence. Although, each sentence is dependent on the lead-in sentence. I'll leave it up to you!

Reply:
What is the "hub"? Does an official name exist?

Reply:
Updated to hub cluster